code/datums/gamemodes/spy_theft.dm

```diff
@@ -721,17 +721,25 @@
 	possible_areas = get_areas_with_unblocked_turfs(/area/station)
 	possible_areas += get_areas_with_unblocked_turfs(/area/diner)
 	possible_areas -= get_areas_with_unblocked_turfs(/area/diner/tug)
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/medical/asylum)			// Donut 3 Asylum
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/security/hos)
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/security/armory)			// Cog1 armory
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/ai_monitored/armory)	// Other map armorys...
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/turret_protected)		// AI secure areas
```

Comment:
this one can be imprecise and seems to cover a lot of areas - its more of an abstract type but i'm cool with removing them

Reply:
I've tried a more accurate one for core and satellite. Still it's very variable on the map design.